### PR TITLE
Wishlist clean

### PR DESCRIPTION
**Pull request by Plumf**
It's all in the description of the commit, a point of vigilance on my last commit or I deleted wishes that didn't seem to me to be free software or that wasn't intended for a server (like web torrent)

There is still an outstanding question as to what to do with all the services that are no longer maintained from 2013 to 2017?

### CODE REVIEW

File: apps_packaging.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Apps packaging in progress
```
Comment (alexAubin) [inline apps_packaging.md]: Hmmm but those apps are expected to be listed in https://yunohost.org/#/apps, though you gotta choose "All apps" to actually see those ... But I'm really not sure we want to add an additional page like this one to maintain on the long term :/

File: apps_wishlist.md [hunk 2]
```diff
@@ -106,10 +97,8 @@ The following list is a compiled wishlist of applications that would be nice-to-
 - [LocomotiveCMS](https://github.com/locomotivecms/engine)
 - [Logstalgia](http://logstalgia.io)
 - [Loomio](https://www.loomio.org)
-- [MagicMirror²](https://github.com/MichMich/MagicMirror)
 - [MaidSafe](http://maidsafe.net)
 - [Mailpile](https://www.mailpile.is)
-- [Mailtrain](https://mailtrain.org/)
```
Comment (alexAubin) [inline apps_wishlist.md]: You sure about removing those ? (Sorry maybe you already answered that question)

Comment (Plumf) [reply apps_wishlist.md]: No, I'm correcting that.

Comment (Plumf) [reply apps_wishlist.md]: https://github.com/YunoHost/doc/pull/1095/commits/16d5221f0ca78fcf2110db1feba9c9d9b909902a

File: apps_packaging.md (hunk 1 above)
Comment (Plumf) [reply apps_packaging.md]: I partly agree with you, that's how I found the links. The proposal results from this [exchange](https://github.com/YunoHost/doc/pull/1056) with @frju365. It was a question for me, and in a longer-term perspective you're right. 

The ultimate solution is to delete these proposals as soon as they are submitted?

Comment (Plumf) [reply apps_packaging.md]: I totally agree with your analysis! I don't realize what this implies at the technical level.

**Comment by Plumf** [comment]
I chose to keep only the modifications of the wishlist page to avoid unnecessary maintenance.

**Comment by Plumf** [comment]
I deleted all apps that had more than a year of no maintenance. And I added links to the rest even if it makes the page heavier it seems more useful for potential packagers

**Pull request merged**